dashboard-app/src/components/StackedBarChart/utils/chartjsUtils.ts

```diff
@@ -25,47 +27,72 @@ export const totalizer = {
   },
 };
 
-export const getStackedGraphOptions = (xAxisTitle: string, yAxisTitle: string): any => ({
-  legend: {
-    display: false,
-  },
-  scales: {
-    xAxes: [{
-      scaleLabel: {
-        display: true,
-        labelString: xAxisTitle,
-      },
-      stacked: true,
-      gridLines: { display: false },
-      ticks: { padding: 5 },
-    }],
-    yAxes: [{
-      scaleLabel: {
-        display: true,
-        labelString: yAxisTitle,
+
+export const getStackedGraphOptions = (
+  xAxisTitle: string,
+  yAxisTitle: string,
+  unit: DurationUnitEnum): any => ({
```

Comment:
Does this need to be `any`?

Reply:
seemed to be something in my package-lock causing me to need this 🤷🏽‍♀️. Gone now...